orca-api/src/main/java/com/netflix/spinnaker/orca/api/pipeline/graph/StageDefinitionBuilder.java

```diff
@@ -108,6 +110,9 @@ default void prepareStageForRestart(@Nonnull StageExecution stage) {}
    */
   static String getType(Class<? extends StageDefinitionBuilder> clazz) {
     String className = clazz.getSimpleName();
+    checkState(
```

Comment:
I'd prefer to see this without the guava dependency. I'm sure its already somehow transitively in orca-api but this is a dependency we strongly want to avoid in API packages so lets not introduce it in this class to save 2 lines of code

Reply:
Good call, I removed it.